charts/prometheus-redis-exporter/templates/servicemonitor.yaml

```
@@ -4,7 +4,9 @@ kind: ServiceMonitor
 metadata:
 {{- if .Values.serviceMonitor.labels }}
   labels:
-{{ toYaml .Values.serviceMonitor.labels | indent 4}}
+    {{- toYaml .Values.serviceMonitor.labels | nindent 4}}
+    {{- end }}
+    {{- toYaml .Values.serviceMonitor.additionalLabels | nindent 4 }}
```

This shall fix the lint error.
```suggestion
    {{- toYaml .Values.serviceMonitor.additionalLabels | nindent 4 }}
    {{- end }}
```